docs/guides/hybrid-graphics.md

```diff
@@ -1,110 +1,61 @@
 # Introduction
 
-This page describes how to use the iGPU on MacBookPro's with Hybrid Graphics (2 GPUs). 13 inch MacBooks only have an iGPU, and do not need this. Using the iGPU means you can save power by putting the more powerful AMD dGPU in a low power state when you don't need it.
+This page describes how to use the iGPU, dGPU, or both. 13 inch MacBooks only have an iGPU, and do not need this. Using the iGPU means you can save power by putting the more powerful AMD dGPU in a low power state when you don't need it.
```

Comment:
This change seems to be unecessary. The previous line seems to be better and more clear.